.vscode/settings.json

```diff
@@ -1,5 +1,8 @@
 {
     "latex-workshop.view.pdf.viewer": "tab",
+    "workbench.editorAssociations": {
+        "*.pdf": "latex-workshop-pdf-hook"
```

Comment:
sicher dass die änderung überhaupt ewas macht? ich hab das gefühl, das passiert so standardmässig schon

Reply:
nein. wenn man im Explorer auf eine PDF Datei klickt, versucht VS Code sie mit dem internen Text-Editor zu öffnen. Dieser fragt, ob man denn wirklich diese Binär-Datei anzeigen will und macht das auf Wunsch auch. Per "Öffnen mit..." kann man die Datei in LaTeX-Workshops PDF-Viewer öffnen (das gleiche was auch passiert, wenn man auf View Latex-PDF geht). Das hier setzt diese Aktion als Standard für alle PDFs

Reply:
geht bei mir auch so. macht aber auch nix kaputt, also w/e.
/resolve